framework/src/modules/token/api.ts

```diff
@@ -52,26 +63,38 @@ export class TokenAPI extends BaseAPI {
 		apiContext: APIContext,
 		senderAddress: Buffer,
 		recipientAddress: Buffer,
-		_tokenID: TokenID,
+		tokenID: TokenID,
```

Comment:
this `transfer` function is the main change. Other API will be properly implemented in #7122